t/lib/test_admin.lua

```diff
@@ -128,6 +128,7 @@ function _M.test(uri, method, body, pattern, headers)
     end
     if not headers["Content-Type"] then
         headers["Content-Type"] = "application/x-www-form-urlencoded"
+        headers["X-API-KEY"] = "TEST_API_KEY"
```

Comment:
random key or hard code key, not using `TEST_API_KEY` as key

Reply:
random key already, please take a look at the new version